superbench/analyzer/file_handler.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""A module for data analysis."""
+
+from pathlib import Path
+import re
+
+import json
+import jsonlines
+import pandas as pd
+import yaml
+
+from superbench.common.utils import logger
+
+
+def read_raw_data(raw_data_path):
+    """Read raw data from raw_data_path and store them in self._raw_data_df.
+
+    Args:
+        raw_data_path (str): the path of raw data jsonl file
+    Returns:
+        DataFrame: raw data, node as index, metric name as columns
+    """
+    p = Path(raw_data_path)
+    raw_data_df = pd.DataFrame()
+    if not p.is_file():
+        logger.error('DataDiagnosis: invalid raw data path - {}'.format(raw_data_path))
+        return raw_data_df
+
+    try:
+        with p.open(encoding='utf-8') as f:
+            for single_node_summary in jsonlines.Reader(f):
+                raw_data_df = raw_data_df.append(single_node_summary, ignore_index=True)
+        raw_data_df = raw_data_df.rename(raw_data_df['node'])
+        raw_data_df = raw_data_df.drop(columns=['node'])
+    except Exception as e:
+        logger.error('Analyzer: invalid raw data fomat - {}'.format(str(e)))
+    return raw_data_df
+
+
+def read_rules(baseline_file=None):
+    """Read rule from rule yaml file.
+
+    Args:
+        baseline_file (str, optional): The path of rule yaml file. Defaults to None.
+
+    Returns:
+        dict: dict object read from yaml file
+    """
+    default_rule_file = Path(__file__).parent / 'default_rule.yaml'
+    p = Path(baseline_file) if baseline_file else default_rule_file
+    if not p.is_file():
+        logger.error('DataDiagnosis: invalid rule file path - {}'.format(str(p.resolve())))
+        return None
+    baseline = None
+    with p.open() as f:
+        baseline = yaml.load(f, Loader=yaml.SafeLoader)
+    return baseline
+
+
+def read_baseline(baseline_file):
+    """Read baseline from baseline json file.
+
+    Args:
+        baseline_file (str): The path of baseline json file.
+
+    Returns:
+        dict: dict object read from json file
+    """
+    p = Path(baseline_file)
+    if not p.is_file():
+        logger.error('DataDiagnosis: invalid baseline file path - {}'.format(str(p.resolve())))
+        return None
+    baseline = None
+    with p.open() as f:
+        baseline = json.load(f)
+    return baseline
+
+
+def excel_raw_data_output(writer, raw_data_df, sheet_name):
```

Comment:
Naming looks a little weird. For util functions, most of functions have verbs at the beginning? Just like `read_baseline`, `read_rules`.
How about `output_excel_raw_data` or `write_excel_raw_data`?

Reply:
Same with `excel_data_not_accept_output`

Reply:
ok, updated to ouptut_excel_xxx